nrm_django/grants/forms.py

```diff
@@ -15,17 +15,28 @@ class MinGrantForm(forms.ModelForm):
     Defines the minimum viable Grant/proposal creation form.
     """
 
+    # Require all fields in minimal grant creation form.
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        for key in self.Meta.fields:
+            self.fields[key].required = True
+        self.fields[
```

Comment:
How is this related to the field definition on grants/models.py L98?

Reply:
L23 onward removes the text "This date cannot be after the Expiration date." from this minimal grant creation form. I removed it because there is no expiration date field on this form, so the reference to it is confusing.